README.md

```diff
@@ -81,9 +81,8 @@ new Stripe(context, "pk_test_yourkey");
 ```
 
 ## Examples
-
+- The [example project](https://github.com/stripe/stripe-android/tree/master/example) demonstrates using our prebuild ui (single-step), prebuild UI (multi-step), card element, and API bindings, including how to create Payment Intents, and Payment Methods; how to use the Stripe class's synchronous and asynchronous methods; and how to use the CardInputWidget.
```

Comment:
Is it prebuild, prebuilt, pre-build, or pre-built? Not sure what is preferred.

Reply:
This will match yuki's documentation.

Reply:
Sg